extras/userdoc/md/documentation/quickref.md

```diff
@@ -174,7 +174,7 @@ Create a nested multi-dimensional network structure in the current subnet.
 
 ### Neuron Models
 
-[iaf\_neuron](../helpindex/cmds/iaf_neuron),
+[iaf\_neuron](../helpindex/cmds/iaf_psc_alpha),
```

Comment:
Could't we delete this line? 

    [iaf\_neuron](../helpindex/cmds/iaf_psc_alpha)